test/conftest.py

```diff
@@ -421,6 +290,8 @@ def get_simple_updater(
     metric=None,
     deterministic_trainer=False,
 ):
+    if learner_kwargs is None:
```

Comment:
Why this form?

Reply:
Yes. Is there a test that populates different `learner_kwargs`

Reply:
yes, e.g., test_joint.py L27. This change is not changing the behavior but rather avoiding the unwanted behavior when using dicts and the likes as defaults in functions.